packages/next/pages/_document.tsx

```diff
@@ -736,6 +736,18 @@ export class NextScript extends Component<OriginProps> {
         {appScript}
         {staticMarkup ? null : this.getDynamicChunks()}
         {staticMarkup ? null : this.getScripts()}
+        {process.env.__NEXT_GRANULAR_CHUNKS ? (
+          <script
+            async
```

Comment:
How did you decide on async instead of defer?

Reply:
I did not see much [difference between them](https://docs.google.com/document/d/1eETKg4eipTNnxcQkQt5UzfwzlSQeAVrfFdJ_3eCBauE)

I kept it consistent with the other async tags. Do you recommend defer instead?